static/js/stream_create.js

```diff
@@ -169,20 +171,34 @@ function create_stream() {
     var stream_name = $.trim($("#create_stream_name").val());
     var description = $.trim($("#create_stream_description").val());
     var is_invite_only = $('#stream_creation_form input[name=privacy]:checked').val() === "invite-only";
+    var is_invite_only_public_history = $('#stream_creation_form input[name=privacy]:checked').val() === "invite-only-public-history";
     var principals = get_principals();
 
+    var invite_only = false;
+    var history_public_to_subscribers = true;
```

Comment:
I would just declare these as `var invite_only` and then set the values in a third `else` clause below.